tests/compose-tests/test-misc-tweaks.sh

```diff
@@ -56,10 +56,35 @@ postprocess:
    set -xeuo pipefail
    test -f /usr/share/included-postprocess-test
 EOF
+
+for x in $(seq 3); do
+  rm tmp/usr -rf
+  mkdir -p tmp/usr/{bin,share}
+  mkdir tmp/usr/share/testsubdir-${x}
+  echo sometest${x} > tmp/usr/bin/sometestbinary-${x}
+  chmod a+x tmp/usr/bin/sometestbinary-${x}
+  echo sometestdata${x} > tmp/usr/share/sometestdata-${x}
+  echo sometestdata-subdir-${x} > tmp/usr/share/testsubdir-${x}/test
+  ostree --repo="${repobuild}" commit --consume --no-xattrs --owner-uid=0 --owner-gid=0 -b testlayer-${x} --tree=dir=tmp
+done
+rm tmp/usr -rf
+mkdir -p tmp/usr/share/info
+echo some info | gzip > tmp/usr/share/info/bash.info.gz
```

Comment:
Don't we already run this test without docs on? Maye instead something we definitely know will exist, e.g. `/usr/bin/ls`?